gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/MySqlDagManagementStateStore.java

```diff
@@ -142,8 +142,11 @@ public void markDagFailed(DagManager.DagId dagId) throws IOException {
 
   @Override
   public void deleteDag(DagManager.DagId dagId) throws IOException {
-    this.dagStateStore.cleanUp(dagId);
-    log.info("Deleted dag {}", dagId);
+    if (this.dagStateStore.cleanUp(dagId)) {
+      log.info("Deleted dag {}", dagId);
+    } else {
+      log.info("Dag deletion was tried but did not happen {}", dagId);
```

Comment:
only `.info` level, not `warn` or even `error`?  alternatively this could arguably be an exception

Reply:
i mean, deletion not happening can really be just because the element was already absent...
if there is some mysql exception, it would be thrown, it will not return a boolean in that case.

Reply:
Should be try/catch in this case with logging?

Reply:
i mean, i intend to not swallow exceptions.
and i am just fine if cleanup returns false. so did not do try/catch